README.md

```diff
@@ -14,15 +14,18 @@ The basics enable many things. Read below.
 These actions can earn you extra weekend time
     
     - Get dad up such that he is working out by 4:30 (+30 Mins)
+    - Exercising on the treadmill for at least 30 Mins to earn (+30 Mins)
+    - Mowing the back or front part of the lawn earns your (+1 hour)
```

Comment:
change to "Mowing the entire lawn (complete with trimming) (+1 hour)" and you have a deal